sweat/algorithms/metrics.py

```diff
@@ -0,0 +1,171 @@
+"""Calculation of performance metrics that change the shape of stream"""
+
+import numpy as np
+import pandas as pd
+from sweat.algorithms.streams import rolling_mean, mask_fill, compute_zones
+from sweat.utils import cast_array_to_original_type
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+def power_duration_curve(arg, mask=None, value=0.0, **kwargs):
```

Comment:
"mean_max_power_curve"?
This algorithm (my slower one) is also in sweat/algorithms/main.py so that has to be deleted. Is metrics a better place do put it?

Reply:
Yes, should be resolved. I've created an issue #9